recipes/lotterycn/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "lotterycn" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lotterycn-{{ version }}.tar.gz
+  sha256: 9d2ff8d2c50f749932d2bf3b13dbf9f2c5686578850fac252d01f14ca4dafe8a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - tqdm >=4.64.1
+    - numpy >=1.21.6
+    - pandas >=1.4.0
+    - requests >=2.28.1
+
+test:
+  imports:
+    - lotterycn
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/uliontse/lotterycn
+  summary: LotteryCN is a library which aims to bring enjoyable analysis of lottery in Python
+  dev_url: https://github.com/UlionTse/lotterycn
```

Comment:
These URLs result in a 404. Is the repository private?

Reply:
@dopplershift I changed it to public, thanks.